src/Tools/dotnet-dump/Commands/SOSCommand.cs

```diff
@@ -0,0 +1,90 @@
+using Microsoft.Diagnostic.Utilities;
+using SOS;
+using System.CommandLine;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostic.Tools.Dump
+{
+    [Command(Name = "sos", Help = "Executes a SOS command.")]
```

Comment:
I think any usage of the sos prefix as a way to dispatch commands should be an internal implementation detail only. I'd advocate that if the user actually types 'sos [anything]' at the shell, they get an error that says 'sos' isn't a command. Seeing help text for it suggested that wasn't the case.